modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextStore.java

```diff
@@ -225,6 +231,180 @@ public DurableStoreConnection init(ConfigurationProperties connectionProperties)
     public void deleteBindingInformation(String exchangeName, String boundQueueName) throws AndesException;
 
     /**
+     * Create a new slot in store
+
+     * @param startMessageId start message id of slot
+     * @param endMessageId end message id of slot
+     * @param storageQueueName name of storage queue name
+     * @param assignedNodeId
+     * @throws AndesException
+     */
+    public void createSlot(long startMessageId, long endMessageId,
```

Comment:
As done before, why not remove the redundant access modifier in this particular interface as well? Always, maintain consistency in your code, which is key to preserve readability a great deal.